server/src/metric-cli.ts

```diff
@@ -80,41 +81,39 @@ export async function run(argv: string[]) {
 
   const github = new GitHub();
 
-  if (args.metric === 'review-latency') {
-    const result =
-        await getReviewLatency(github, {org: args.org, repo: args.repo});
-    if (!args.raw) {
-      console.info(result.format());
-    } else {
-      result.logRawData();
-    }
+  const metricSpinner = ora(`Gathering '${args.metric}' metric data`).start();
 
-  } else if (args.metric === 'issue-counts') {
-    const counts =
-        await getIssueCounts(github, {org: args.org, repo: args.repo});
-    if (args.raw) {
-      for (const point of counts.timeSeries()) {
-        console.log([point.date, point.numOpened, point.numClosed].join('\t'));
-      }
-    } else {
-      console.info(counts.summary());
-    }
-  } else if (args.metric === 'review-coverage') {
-    const yearAgo = new Date();
-    yearAgo.setFullYear(yearAgo.getFullYear() - 1);
-    const result = await getReviewCoverage(
-        github, {org: args.org, repo: args.repo, since: yearAgo.toISOString()});
-    // TODO: Implement a raw API.
-    if (!args.raw) {
-      console.log(result.summary());
-    }
-  } else if (args.metric === 'stars') {
-    const stars = await getStars(github, {org: args.org, repo: args.repo});
-    // TODO: Implement a raw API.
-    if (!args.raw) {
-      console.log(stars.summary());
-    }
+  let metricResult;
+  let orgInfo = {org: args.org, repo: args.repo};
+  switch (args.metric) {
+    case 'review-latency':
+      metricResult = await getReviewLatency(github, orgInfo);
```

Comment:
Fine for now.

In a follow up PR, we could maybe change this to be a map of string to class and then maybe a `metric.compute()` function.

Reply:
👍